app/src/main/res/layout/profile_edit_activity.xml

```diff
@@ -12,228 +12,236 @@
       type="org.oppia.android.app.settings.profile.ProfileEditViewModel" />
   </data>
 
-  <androidx.constraintlayout.widget.ConstraintLayout
+  <ScrollView
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/profileEditBackground"
-    android:clipToPadding="false">
+    android:layout_margin="0dp"
+    android:fillViewport="true"
+    android:padding="0dp">
 
-    <LinearLayout
-      android:id="@+id/linearLayout"
+    <androidx.constraintlayout.widget.ConstraintLayout
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:clipToPadding="false"
-      android:orientation="vertical"
-      app:layout_constraintEnd_toEndOf="parent"
-      app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toBottomOf="@id/profile_edit_main">
-
-      <View
-        android:layout_width="match_parent"
-        android:layout_height="1dp"
-        android:background="@color/grey" />
-
-      <Button
-        android:id="@+id/profile_rename_button"
-        style="?android:attr/borderlessButtonStyle"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:background="@color/white"
-        android:fontFamily="sans-serif"
-        android:gravity="start|center_vertical"
-        android:minHeight="48dp"
-        android:paddingStart="16dp"
-        android:paddingEnd="16dp"
-        android:text="@string/profile_edit_rename"
-        android:textAllCaps="false"
-        android:textColor="@color/oppiaPrimaryText"
-        android:textSize="16sp" />
-
-      <View
-        android:layout_width="match_parent"
-        android:layout_height="1dp"
-        android:background="@color/grey" />
-
-      <Button
-        android:id="@+id/profile_reset_button"
-        style="?android:attr/borderlessButtonStyle"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:background="@color/white"
-        android:fontFamily="sans-serif"
-        android:gravity="start|center_vertical"
-        android:minHeight="48dp"
-        android:paddingStart="16dp"
-        android:paddingEnd="16dp"
-        android:text="@string/profile_edit_reset"
-        android:textAllCaps="false"
-        android:textColor="@color/oppiaPrimaryText"
-        android:textSize="16sp" />
-
-      <View
-        android:layout_width="match_parent"
-        android:layout_height="1dp"
-        android:background="@color/grey" />
+      android:background="@color/profileEditBackground"
+      android:clipToPadding="false">
 
       <LinearLayout
+        android:id="@+id/linearLayout"
```

Comment:
While you're here, can you fix this ID to either be more descriptive about what it contains (see other layout IDs for references), or remove it if nothing's referencing the ID?

Reply:
changed to profile_edit_actions_container